webpack/set-webpack-public-path-plugin/src/codeGenerator.ts

```diff
@@ -45,54 +47,77 @@ export function getSetPublicPathCode(options: IInternalOptions, emitWarning: (wa
     }
 
     lines.push(...[
-      'var found = false;',
+      `var ${varName};`,
       '',
       'if (scripts && scripts.length) {',
       '  for (var i = 0; i < scripts.length; i++) {',
       '    if (!scripts[i]) continue;',
       `    var path = scripts[i].getAttribute('src');`,
       '    if (path && path.match(regex)) {',
-      `      ${options.webpackPublicPathVariable} = path.substring(0, path.lastIndexOf('/') + 1);`,
-      '      found = true;',
+      `      ${varName} = path.substring(0, path.lastIndexOf('/') + 1);`,
       '      break;',
       '    }',
       '  }',
       '}',
       '',
-      'if (!found) {',
+      `if (!${varName}) {`,
       `  for (var global in ${registryVariableName}) {`,
       '    if (global && global.match(regex)) {',
-      `      ${options.webpackPublicPathVariable} = global.substring(0, global.lastIndexOf('/') + 1);`,
+      `      ${varName} = global.substring(0, global.lastIndexOf('/') + 1);`,
       '      break;',
       '    }',
       '  }',
       '}'
     ]);
+
+    if (options.getPostProcessScript) {
+      lines.push(...[
+        '',
+        `if (${varName}) {`,
+        `  ${options.getPostProcessScript(varName)};`,
```

Comment:
It's not a function, it's a code snippet. This function returns a block of script as text that modifies the variable, not an actual function.